settings/installer/ospd/ospd.spec

```diff
@@ -42,13 +55,16 @@ subparsers:
                     help: The overcloud storage type
                     default: ceph.yml
 
-            - title: Overcloud images
+            - title: Product images
               options:
                 images-task:
                     type: Value
                     help: Specifies whether the images should be built or imported
                     required: yes
                     choices: [import, build]
+                images-url:
```

Comment:
this is a little problematic.. the "--images-url" is only necessary when doing an import, not when doing a build.
I don't mind it being here, we just need to think of how to remove it in the future as it might confuse people